docker/sample_data/timescale_example.sql

```diff
@@ -0,0 +1,191 @@
+CREATE TABLE public.product (
+    id INT PRIMARY KEY,
+    name CHARACTER VARYING(100),
+    price MONEY
+);
+
+CREATE TABLE public.address (
+    id BIGINT PRIMARY KEY,
+    house INT,
+    street CHARACTER VARYING(100),
+    city CHARACTER VARYING(100),
+    state CHARACTER VARYING(100),
+    zip CHARACTER VARYING(100)
+);
+
+CREATE TABLE public.customer (
+    id INT PRIMARY KEY,
+    email CHARACTER VARYING(100),
+    name  CHARACTER VARYING(100),
+    created TIMESTAMP,
+    address_id BIGINT
+);
+
+CREATE TABLE public.employee (
+    id INT PRIMARY KEY,
+    email CHARACTER VARYING(100),
+    name CHARACTER VARYING(100),
+    address_id BIGINT
+);
+
+CREATE TABLE public.payment_card (
+    id CHARACTER VARYING(100) PRIMARY KEY,
+    name CHARACTER VARYING(100),
+    ccn BIGINT,
+    code SMALLINT,
+    preferred BOOLEAN,
+    customer_id INT,
+    billing_address_id BIGINT
+);
+
+CREATE TABLE public.orders (
+    id CHARACTER VARYING(100) PRIMARY KEY,
+    customer_id INT,
+    shipping_address_id BIGINT,
+    payment_card_id CHARACTER VARYING(100)
+);
+
+CREATE TABLE public.order_item (
+    order_id CHARACTER VARYING(100),
+    item_no SMALLINT,
+    product_id INT,
+    quantity SMALLINT,
+    CONSTRAINT order_item_pk PRIMARY KEY (order_id, item_no)
+);
+
+CREATE TABLE public.visit (
+    email CHARACTER VARYING(100),
+    last_visit TIMESTAMP,
+    CONSTRAINT visit_pk PRIMARY KEY (email, last_visit)
+);
+
+CREATE TABLE public.login (
+    id INT PRIMARY KEY,
+    customer_id INT,
+    time TIMESTAMP
+);
+
+CREATE TABLE public.service_request (
+    id CHARACTER VARYING(100) PRIMARY KEY,
+    email CHARACTER VARYING(100),
+    alt_email CHARACTER VARYING(100),
+    opened DATE,
+    closed DATE,
+    employee_id INT
+);
+
+CREATE TABLE public.report (
+    id INT PRIMARY KEY,
+    email CHARACTER VARYING(100),
+    name CHARACTER VARYING(100),
+    year INT,
+    month INT,
+    total_visits INT
+);
+
+CREATE TABLE public.composite_pk_test (
+    id_a INT NOT NULL,
+    id_b INT NOT NULL,
+    description VARCHAR(100),
+    customer_id INT,
+    PRIMARY KEY(id_a, id_b)
+);
+
+CREATE TABLE public.type_link_test (
+    id CHARACTER VARYING(100) PRIMARY KEY,
+    name CHARACTER VARYING(100)
+);
+
+CREATE TABLE public.onsite_personnel (
+    responsible CHARACTER VARYING(100),
+    time TIMESTAMP PRIMARY KEY
+);
+
+
+INSERT INTO public.onsite_personnel VALUES
+('employee-1@example.com', '2022-01-01 09:00:00'),
+('employee-1@example.com', '2022-01-02 09:00:00'),
+('employee-1@example.com', '2022-01-03 09:00:00'),
+('employee-2@example.com', '2022-01-04 09:00:00'),
+('employee-1@example.com', '2022-01-05 09:00:00'),
+('employee-2@example.com', '2022-01-06 09:00:00'),
+('employee-2@example.com', '2022-01-07 09:00:00'),
+('employee-2@example.com', '2022-01-08 09:00:00');
+
+SELECT create_hypertable('onsite_personnel', 'time', migrate_data => true);
```

Comment:
[Creates a hypertable here](https://docs.timescale.com/timescaledb/latest/how-to-guides/hypertables/create/#create-a-hypertable)  by first creating a postgres table and then converting it into a hypertable.  You have to specify the column that holds the "time" values. In this contrived example, I have unique times that should be the primary key with a "responsible" employee who was on call at that time.

I've added an integration test that demonstrates a basic access and erasure request on this hypertable.  Some comments are that we require primary keys to erase. Noting that timescale requires the unique/primary key dimension to contain the `time` dimension.  You can have composite primary keys including the time dimension in Timescale and fidesops has support for this.

Also worth noting that "datetimes" are not a fully supported date type in Fidesops yet.  A lot of our existing examples that deal with dates tend to treat dates as strings.  Masking date fields can be problematic outside the null masking strategy.